packages/components/src/DataList/components/DataListHeaderTile/components/DataListSortingOptions.css

```diff
@@ -0,0 +1,44 @@
+.optionsList {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  max-width: var(--popover--width);
+  box-shadow: var(--shadow-base);
+  margin-top: var(--space-smaller);
+  padding: var(--space-small) 0;
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-larger);
+  background: var(--color-surface);
+  align-items: flex-start;
+}
+
+.option {
+  display: flex;
+  min-width: calc(var(--space-largest) * 3.5);
+  min-height: calc((var(--space-largest) - var(--space-smaller)));
+  box-sizing: border-box;
+  margin: 0 var(--space-small);
+  padding: var(--space-small);
+  font-family: var(--typography--fontFamily-normal);
+  font-size: var(--typography--fontSize-base);
+  font-weight: 500;
+  white-space: nowrap;
+  list-style: none;
+  cursor: pointer;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.option:hover,
+.option:focus {
+  border-radius: var(--radius-large);
```

Comment:
As a favour can I please get the retheme radius specified on this one as well (example [here](https://github.com/GetJobber/atlantis/blob/7c2af0111b3ee141b35861872f7fdf7f1aca3c57/packages/components/src/Combobox/components/ComboboxOption/ComboboxOption.css#L15-L18))

Reply:
We moved `border-radius` to option here: ee1423098f7ad2d4250e87076edf50478938b89c and set the retheme radius here: cd2b3022ed9844e16ec123391e6a68a12dd6380b